Automattic/wp-calypso
# Jetpack Connect: Add Agency marketing above and below the prices

#### Proposed Changes

This PR makes 2 small changes to the [Jetpack Connect pricing page](https://wordpress.com/jetpack/connect/store) to promote Jetpack for Agencies

**Change 1**

Add a third column to the intro pricing banner with a link to the [Agencies page](https://jetpack.com/for/agencies/) on Jetpack.com

**Change 2**

Add another button in the "Need more info?" section below the featured pricing blocks that also leads to the Agencies page

**Notes**

* Both of these links have tracks events set up, the names are listed below
   - `calypso_jpcom_agencies_page_more_info_button_link_click`
   - `calypso_jpcom_agencies_page_intro_banner_link_click`

#### Testing Instructions

1. Checkout this branch via `git switch add/agency-marketing-to-jetpsack-connect`
2. Start your local environment via `yarn start`
3. Go to http://calypso.localhost:3000/jetpack/connect/store
4. Confirm there is a new link in the intro banner and confirm that it goes to the [Agencies page](https://jetpack.com/for/agencies/)
![Screen Shot 2022-07-06 at 1 41 23 PM](https://user-images.githubusercontent.com/65001528/177639455-abb21853-6cda-4792-8839-495f694c66a5.jpg)
5. Scroll down below the featured pricing blocks and confirm the "Need more info?" box has a new button that leads to the [Agencies page](https://jetpack.com/for/agencies/)
![Screen Shot 2022-07-06 at 1 44 05 PM](https://user-images.githubusercontent.com/65001528/177639733-4c14ffb4-ca28-4eec-9b31-4edde1bb84c0.jpg)

**Full view**
![Screen Shot 2022-07-06 at 1 47 25 PM](https://user-images.githubusercontent.com/65001528/177640325-42a6374a-2c96-4e61-a83f-2bfb5cd245a8.jpg)

**Testing the tracks**
1. Click on the agencies link in the intro banner
2. Wait roughly 5 minutes and go to the Tracks Live View in MC and search for `calypso_jpcom_agencies_page_intro_banner_link_click`  to confirm the event was tracked
3. Click on the new button under "Need more info?"
4. Wait roughly 5 minutes and go to the Tracks Live View in MC and search for `calypso_jpcom_agencies_page_more_info_button_link_click` to confirm the event was tracked


**Bonus**
The agencies link in the intro banner should disappear on the sticky banner on mobile as it took up too much space on the screen to have 3 lines of stacked text. You can use the dev tools to look at a mobile view and confirm this behavior
